cli/activate.go

```diff
@@ -110,29 +108,29 @@ func activate(c *cli.Context) error {
 	destFolder := path
 
 	if exist, err := pathutil.IsPathExists(destFolder); err != nil {
-		log.Fatalln("[STEPMAN] - Failed to check path:", err)
+		log.Fatalf("Failed to check path, error: %s", err)
 	} else if !exist {
 		if err := os.MkdirAll(destFolder, 0777); err != nil {
-			log.Fatalln("[STEPMAN] - Failed to create path:", err)
+			log.Fatalf("Failed to create path, error: %s", err)
 		}
 	}
 
 	if err = command.CopyDir(srcFolder+"/", destFolder, true); err != nil {
-		log.Fatalln("[STEPMAN] - Failed to copy step:", err)
+		log.Fatalf("Failed to copy step, error: %s", err)
 	}
 
 	// Copy step.yml to specified path
 	if copyYML != "" {
 		if exist, err := pathutil.IsPathExists(copyYML); err != nil {
-			log.Fatalln("[STEPMAN] - Failed to check path:", err)
+			log.Fatalf("Failed to check path, error: %s", err)
 		} else if exist {
-			log.Fatalln("[STEPMAN] - Copy yml destination path exist")
+			log.Fatalf("Copy yml destination path exist")
```

Comment:
`Copy yml destination path exist` -> `Failed to copy step.yml, error: destination path exists`